base/libgit2/types.jl

```diff
@@ -705,13 +705,19 @@ function getobjecttype(obj_type::Cint)
     end
 end
 
+import Base.securezero!
+
 "Credentials that support only `user` and `password` parameters"
 type UserPasswordCredentials <: AbstractCredentials
     user::AbstractString
```

Comment:
if this were ever using a different type of string, securezero would fail with a methoderror, right?